content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc

```diff
@@ -0,0 +1,175 @@
+
+
+=== Overview of Procedure
+
+The configuration steps to be performed after you have inserted the primary factory preset YubiHSM 2 device into the air-gapped system are set out briefly below. These steps are described in detail in the subsequent "Procedure: Configuring Primary YubiHSM 2."
+
+1. Set up communication between the YubiHSM 2 tools and the device.
+
+2. Start the configuration process.
+
+3. Authenticate to the YubiHSM 2 device.
+
+4. Specify whether you need to add RSA decryption.
+
+5. Enter the name of the domain in which you need the application authentication key and audit key to be available in.
+
+6. Create the wrap key. The wrap key is very important as it allows you to export and import objects from and to the device. For example, you would export and import objects for backup purposes, as described in https://Backup_Key_Material.html[Backup Key Material].
+
+7. Split the wrap key into shares and record the number of shares required to regenerate the wrap key.
+
+8. Create the application authentication key, which is used to authenticate to the YubiHSM 2 device through the KSP. The password to the authentication key must be at least eight (8) characters.
+
+9. Create the audit key to access the internal audit log of the device and to reset the audit log. The device audit log retains information about the last 62 operations.
+
+image::yubi-hsm-2-flowchart.png[]
+
+**Figure 1 – Flowchart illustrating the YubiHSM 2 setup for Windows**
+
+=== Procedure: Configuring Primary YubiHSM 2
+
+*Step 1*: Enable communication with the YubiHSM 2 device by running the YubiHSM Connector on the system where the device is inserted. If the YubiHSM Connector is running on a host machine to which the YubiHSM 2 is physically connected, the Connector should be started in networked mode. For example, if the host IP address is 192.168.100.252, the Connector should be started on the host machine with the following command:
+
+        $ yubihsm-connector -l 192.168.100.252:12345
+
+In this scenario, you can validate that the Connector is running properly by typing the following URL into your web browser:
+
+        http://192.168.100.252:12345/connector/status
+
+The output in the web browser should be similar to:
+
+        status=OK
+        serial=*
+        version=1.0.0
+        pid=*
+        address=192.168.100.252
+        port=12345
+
+*Step 2*: In the Command Prompt, navigate to a directory for which you have write access and run the YubiHSM Setup with the argument `ksp`.
+
+        $ yubihsm-setup ksp
+
+_Note: When replacing the default authentication key using this setup, the new authentication key will only be operable in the same domain as the asymmetric key. This is due to the https://developers.yubico.com/YubiHSM2/Concepts/Domain.html[domain concept] that is used to compartmentalize the YubiHSM 2 device._
```

Comment:
It's good if the link to Domain.html is relative here, just put it as the relative link "link:../../Concepts/Domain.adoc" and it'll work both in asciidoc mode and html.

Reply:
Done